defaults/main.yml

```diff
@@ -158,7 +159,7 @@ atom_es_fields_limit: "3000"
 
 atom_pool_user: "{{ atom_user }}"
 atom_pool_group: "{{ atom_group }}"
-atom_pool_listen: "/var/run/php-fpm.atom.sock"
+atom_pool_listen: "/var/run/php-fpm.{{ atom_path | basename }}.sock"
```

Comment:
Does it makes sense to run a different php worker for each site?

We are running them all under the same uid (www-data), so not sure of what's the vantage of having more php-fpm sockets.

Other than that, this pr is awesome! I'll give it a try on a Centos server just to verify that everything works.

Kudos @hakamine !

Reply:
I think it's better to have a separate socket for each pool, from a security standpoint. In order to have separate user for each pool, we can override the pool user/group variables in each of the site yml config files.